fleet-management-spring-mvc/pom.xml

```diff
@@ -19,11 +19,6 @@
             <artifactId>fleet-management-sample-data</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
-        <dependency>
-            <groupId>cz.fi.muni.pa165</groupId>
-            <artifactId>fleet-management-rest</artifactId>
-            <version>${project.parent.version}</version>
```

Comment:
how you want to run rest and spring mvc module at the same time when you remove this dependency ?